test/tx_build/amount_test.exs

```diff
@@ -12,6 +14,10 @@ defmodule Stellar.TxBuild.AmountTest do
     %Amount{amount: 3.141516, raw: 31_415_160} = Amount.new(3.141516)
   end
 
+  test "new/2 max" do
+    %Amount{raw: @max} = Amount.new(:max)
```

Comment:
Please match the `amount:` value since it is the result of a division of integers, confront the result of the division in Elixir vs. other official libraries.